app/Http/Controllers/Admin/SessionCrudController.php

```diff
@@ -109,6 +110,8 @@ protected function setupCreateOperation()
         CRUD::field('adventure_id');
         CRUD::field('dungeonMaster');
         CRUD::field('table');
+        CRUD::field('seats')->default(8);
```

Comment:
big table

Reply:
(not a change request)

Reply:
yeah it the typical maximum size that DnD MTL use at their events, most tables dont go that big but i figured it was the number that made the most sense